lib/new_relic/traced_thread.rb

```diff
@@ -7,19 +7,18 @@ module NewRelic
   #
   # This class allows the current transaction to be passed to a Thread so that nested segments can be created from the operations performed within the Thread's block.
   # To have the New Relic Ruby agent automatically trace all of your applications threads,
-  # enable the `instrumentation.thread.tracing` configuration option in your newrelic.yml.
+  # enable the +instrumentation.thread.tracing+ configuration option in your newrelic.yml.
   #
-  # Note: disabling the configuration option `instrumentation.thread` while using this class can cause incorrectly nested spans.
+  # Note: disabling the configuration option +instrumentation.thread+ while using this class can cause incorrectly nested spans.
   #
   # @api public
   class TracedThread < Thread
     #
     # Creates a new Thread whose work will be traced by New Relic.
     # Use this class as a replacement for the native Thread class.
-    # Example: Instead of using `Thread.new`, use:
-    # ```ruby
-    #   NewRelic::TracedThread.new { execute_some_code }
-    # ```
+    # Example: Instead of using +Thread.new+, use:
+    #
```

Comment:
I thought this block needed the ``` ruby``` to designate the code as ruby. Is that no longer needed?

Reply:
It actually wasn't needed originally, I had just done this incorrectly. I thought it was using markdown which does use that, but turns out it doesn't actually use markdown! Yard uses a totally different syntax for documentation, so that is what this change is using instead. So instead of doing a code block like ` ```ruby `, a code block is supposed to just be indented. and instead of doing a snippet with `, you use +, like
```ruby
    ```ruby
    # wow a code block
    this = code.wow
    ```
    # VS 
         # wow a code block
         this = code.wow


    `code snippet`
    # VS
    +code snippet+
```